utils/pre-swap-checker/contracts/PreSwapChecker.sol

```diff
@@ -10,6 +10,7 @@ import "@airswap/swap/contracts/Swap.sol";
 import "@airswap/transfers/contracts/TransferHandlerRegistry.sol";
 import "@airswap/tokens/contracts/interfaces/IWETH.sol";
 import "@airswap/wrapper/contracts/Wrapper.sol";
```

Comment:
What is Wrapper.sol imported for? Can't find it anywhere in the contract

Reply:
Also where the contract uses `Swap(...)` can it not just use `ISwap(...)`? e.g.
`TransferHandlerRegistry tokenRegistry = Swap(swap).registry();`

Reply:
use of registry was never put into ISwap, only in Swap. maybe want to remedy that now.

Reply:
actually unable to do that as TransferHandlerRegistry is a contract not an interface

Reply:
i can be consistent and just use Swap everything and not import both Swap and ISwap though.

Reply:
never mind, ive resolved.